bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java

```diff
@@ -36,6 +37,7 @@ public IOExceptionWithStatus(String message, OperationRuntimeException cause) {
   /**
    * Status from the provided OperationRuntimeException.
    */
+  @SuppressWarnings("deprecation")
```

Comment:
I would put some comment here on why it is ok to ignore the deprecation warning. Warnings are there for a reason, so we should have a reason to suppress them, or talk to grpc if the warning is wrong.

Asides from this and the MS/MILLIS comment, LGTM.